cmd/nitrite/main.go

```diff
@@ -20,12 +20,7 @@ var (
 func main() {
 	flag.Parse()
 
-	if "" == *fAttestation {
-		flag.PrintDefaults()
-		os.Exit(1)
-	}
-
-	if *fAttestation == "-" {
+	if *fAttestation == "" {
```

Comment:
Slight update so that an empty attestation string (i.e., not explicitly using the `-attestation` flag to provide a string) defaults to reading from stdin. I think it makes the use syntax cleaner
```bash
# Old usage
cat ./internal/testdata/nitro_attestation.b64 | go run ./cmd/nitrite -attestation -
cat ./internal/testdata/nitro_debug_attestation.b64 | go run ./cmd/nitrite -attestation - -allowdebug true

# New usage
cat ./internal/testdata/nitro_attestation.b64 | go run ./cmd/nitrite
cat ./internal/testdata/nitro_debug_attestation.b64 | go run ./cmd/nitrite -allowdebug true

# You can still provide the attestation string via the flag if you wish
go run ./cmd/nitrite -attestation yourattestationstringhere
```